tests/console/consoletest_setup.pm

```diff
@@ -100,6 +103,12 @@ sub run() {
         assert_script_run("grub2-mkconfig -o /boot/grub2/grub.cfg");
     }
 
+    # https://trello.com/c/Y57ujlSn/663-3-plan-b-for-sle-12-sp2-p1-988157-hps-bug-sles12-sp2-install-gets-stuck-initializing-on-a-8-blade-hpe-bl920-gen-9-system
+    if (check_var('NETWORK_INIT_PARAM', 'ifcfg=eth0=dhcp')) {
+        # grep all also compressed files e.g. y2log-1.gz
+        assert_script_run "less /var/log/YaST2/y2log*|grep \"Automatic DHCP configuration not started - an interface is already configured\"";
```

Comment:
aaah, now I understand :-)

That would be `zgrep` what you are looking for. Try

```perl
assert_script_run 'zgrep /var/log/YaST2/y2log* "Automatic DHCP configuration not started - an interface is already configured"';
```

With the single ticks you don't need to escape the inner double ticks.